src/modules/Header/index.js

```diff
@@ -17,8 +17,8 @@ const StyledWrapper = styled.div`
   `)};
 `;
 
-const Header = () => (
-  <StyledWrapper>
+const Header = ({ ...props }) => (
```

Comment:
Did you not mean the following?
```suggestion
const Header = props => (
```

Reply:
Good point.. 😄